core/translate.rs

```diff
@@ -763,7 +764,50 @@ fn translate_condition_expr(
             op,
             rhs,
             escape,
-        } => {}
+        } => {
+            let cur_reg = program.alloc_register();
+            assert!(match rhs.as_ref() {
+                ast::Expr::Literal(_) => true,
+                _ => false,
+            });
+            match op {
+                ast::LikeOperator::Like => {
+                    let pattern_reg = program.alloc_register();
+                    let column_reg = program.alloc_register();
+                    // LIKE(pattern, column). We should translate the pattern first before the column
+                    let _ = translate_expr(program, select, rhs, pattern_reg)?;
+                    program.mark_last_insn_constant();
+                    let _ = translate_expr(program, select, lhs, column_reg)?;
+                    program.emit_insn(Insn::Function {
+                        func: SingleRowFunc::Like,
+                        start_reg: pattern_reg,
+                        dest: cur_reg,
+                    });
+                }
+                ast::LikeOperator::Glob => todo!(),
+                ast::LikeOperator::Match => todo!(),
+                ast::LikeOperator::Regexp => todo!(),
+            }
+            if jump_if_true || *not {
```

Comment:
Hmm, isn't the logic so that we want to invert the instruction if `!jump_if_true`, so then it would mean:

```
jump_if_true && not --> don't invert -> Insn::IfNot
jump_if_true && !not --> don't invert -> Insn:If
!jump_if_true && not -> invert -> Insn::If
!jump_if_true && !not -> invert -> Insn::IfNot
```
```

Reply:
would be in favor of renaming `jump_if_true` to `invert_instruction` (and flipping the boolean at the call sites) but that's maybe a separate discussion

Reply:
> so then it would mean:
> jump_if_true && not --> don't invert -> Insn::IfNot
> jump_if_true && !not --> don't invert -> Insn:If
> !jump_if_true && not -> invert -> Insn::If
> !jump_if_true && !not -> invert -> Insn::IfNot

isnt this the same as my condition 😅

Reply:
my bad 😓